teaspoons-cli/.env

```diff
@@ -0,0 +1,10 @@
+# Teaspoons API URL
+TEASPOONS_API_URL=https://tsps.dsde-dev.broadinstitute.org
```

Comment:
i assume at some point we'll want to parameterized this so it can run on different environments

Reply:
yes, added to AC for TSPS-321